ChangeLog

```diff
@@ -1,5 +1,139 @@
-### 3.9.1dev <- NOTE: the release version number will be 3.9.2 ###
+### 3.9.1dev <- NOTE: the release version number will be 3.10.0 ###
 
+- Build: Updated bundled Qt6 to version 6.4.3 (#3025).
+  (contributed by @app/github-actions)
+
+- Mac: Enable dark-mode support on non legacy builds (#2833).
+  (contributed by @ann0see)
+
+- Build: Bump DoozyX/clang-format-lint-action from 0.15 to 0.16 (#3053).
+  (contributed by @app/dependabot)
+
+- Autobuild: Hardened detection of QT 5 builds for macOS and iOS (#3057).
+  (contributed by @ann0see)
+
+- Build: Windows 64bit releases use Qt 6.4.3 now. Jamulus will from now on only support Windows 10 and above (#2300).
+  (contributed by @hoffie)
+
+- Translation: Document translation process shift to Weblate. Translators can from now one use Weblate for translations (#2880).
+  (contributed by @ann0see)
+
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
```

Comment:
This needs to say which platform is affected.

Reply:
And eventually merge all the bumps.

Reply:
Yes. I believe that's something for close to the release.